chaoscenter/authentication/pkg/utils/sanitizers.go

```diff
@@ -23,10 +23,15 @@ func ValidateStrictPassword(input string) error {
 	if len(input) < 8 {
 		return fmt.Errorf("password is less than 8 characters")
 	}
+
+	if len(input) > 16 {
```

Comment:
Upper limit check may not be required

Reply:
Can you explain why that would not be required @SarthakJain26 ?

Reply:
It was just a suggestion. user can enter whatever password he wants until its secure. But its ok to have the check.